src/riscv.h

```diff
@@ -143,10 +144,10 @@ riscv_word_t rv_get_reg(struct riscv_t *, uint32_t reg);
 void syscall_handler(struct riscv_t *rv);
 
 /* environment call handler */
-void ecall_handler(struct riscv_t *rv);
+void ecall_handler(struct riscv_t *rv, struct rv_insn_t *ir);
```

Comment:
Avoid changing the function prototype of `ecall` and `ebreak` handlers.

Reply:
Adding a new parameter in exception handler since the exception default handler needs the length of current instruction executed. Maybe, I think retaining the `insn_len` in `struct riscv_t` to store the current instruction length cans avoid to add a new parameter in all exception handler.

Reply:
Instead of naming it `insn_len`, we can call it `bool compressed`.